### DISCUSSION TITLE
Power Spike in Energy Stats

### DISCUSSION THREAD

**Discussion by Captain0Kangaroo** [macxq/foxess-ha#15] [General]
Not sure in this is an issue as such but this morning the FoxCloud website went down a few times and therefore data was not pulling into HA. Once it was all backup online there was a massive spike in my stats for that period. 
<img width="1159" alt="Screenshot 2021-12-21 at 10 49 50" src="https://user-images.githubusercontent.com/96144685/146917873-5883ee76-d533-445b-a602-5c0f0f12decb.png">
<img width="1157" alt="Screenshot 2021-12-21 at 10 49 28" src="https://user-images.githubusercontent.com/96144685/146917884-2a88718e-465f-4dff-81e6-5b59bd46de67.png">

Did anyone else this this morning?

**Comment by macxq** [comment]
@Captain0Kangaroo  do you see similar spike over the FoxCloud website  ?

**Reply by Daibutt** [reply]
@Captain0Kangaroo @macxq 
I think this may possibly be a combination of Fox Cloud/HA Energy dashboard issue rather than foxess-ha specifically.
I also see spikes in reported battery usage this morning which are not real;
![image](https://user-images.githubusercontent.com/69276073/146927531-3eb0b739-17c8-40b9-9e1c-cf368f2502ba.png)

But if I look at the battery discharge totaliser it dips to zero a couple of times (presumably when Fox Cloud site comes back up?).
![image](https://user-images.githubusercontent.com/69276073/146928104-e96cbeda-30f1-45ab-8c1c-f3d12edafb5a.png)

When this next updates to the correct daily total, HA sees this as actual "consumption" and we get the spikes.

What I am not sure about is if Fox Cloud is generating the zero values, HA is generating zero values or if foixess-ha is the culprit.
My inclination is Fox Cloud or HA as I have also seen this previously with grid elec & gas consumption when having connectivity issues with my meter reading app/software.
